api/docker-compose.yaml

```diff
@@ -36,11 +36,11 @@ services:
       - elastic
     build:
       context: ./src/dev/server
-    user: ${UID}
+    user: ${USER_ID}
     working_dir: /w/api
     volumes:
-      - ..:/w:cached
-      - gradle-cache:/.gradle
+      - src-sync:/w:nocopy
+      - gradle-cache:/.gradle:cached
```

Comment:
What does this do, and why should we keep half of these lines as `gradle-cache:/.gradle` ?

Reply:
Thanks for checking on this one – I did a bit more benchmarking and it has no impact on startup time. Removed.